caches/infinispan/src/test/resources/bmunit/common/WriteDeleteAndVerifyNonExistence.btm

```diff
@@ -0,0 +1,21 @@
+RULE service deleteFile
```

Comment:
So, this set of rules forces the write to complete before delete can proceed, is that true? What happens if you trigger both rules AT ENTRY, such that it's trying to delete and write simultaneously?